chess-AI/train.py

```diff
@@ -260,35 +249,68 @@ def main():
     device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
     nnet = PlayNetwork().to(device=device)
 
-    nnet, dataset_path, stockfish_options, mcts_options, start_train, show_dash = init_params(nnet, device)
-
-    if start_train:
-        msg = "Stockfish Training Has Begun"
-        if show_dash:
-            Dashboard.info_message("success", msg)
-        else:
-            print(msg)
-        train_on_stockfish(nnet, dataset_path, stockfish_options, show_dash)
+    make_dataset_flag, stockfish_train_flag, mcts_train_flag = True, True, True
```

Comment:
These should be returned from `init_params` and added as flags to the argparser. If we don't add them to the dashboard in this PR (which I would prefer we do), let's default to setting them true if `args.dashboard` is `True`. This would affect `train.sh` so we could update that as well to handle these additional CLI arguments.